src/components/collection_voting_slab/MultiDatasetVoteSlab.tsx

```diff
@@ -0,0 +1,254 @@
+    td: {
+      padding: '10px 10px 20px 20px'
+    }
+  },
+  slabTitle: {
+    display: 'flex',
+    paddingBottom: '15px',
+  },
+  slatTitleText: {
+    display: 'flex',
+    fontSize: 17,
+    fontWeight: 800,
+    height: '32px',
+    paddingLeft: '-10%',
+    color: '#333F52',
+    marginTop: '-5px',
+    columnGap: '2rem'
+  },
+  question: {
+    fontSize: 17,
+    color: '#333F52',
+    marginLeft: '30px',
+  },
+  dataUses: {},
+  voteInfo: {},
+  chairVoteInfo: {},
+};
+
+// Components
+const DataUseSummary = ({ bucket }: DataUseSummaryProps) => {
+  const dataUses = get(bucket, 'dataUses', []);
+  return !isNil(dataUses) 
+    ? <div style={styles.dataUses}>{DataUsePills(dataUses)}</div>
+    : <></>;
+};
+
+const VoteInfoSubsection = ({
+  currentUserVotes,
+  bucket,
+  isChair,
+  isApprovalDisabled,
+  isLoading,
+  readOnly,
+  adminPage,
+  updateFinalVote,
+  reloadFn,
+}: VoteInfoSubsectionProps) => {
+  const electionIds = currentUserVotes.map((vote) => vote.electionId);
+  const allOpenElections = bucket.elections
+    .filter((election) => electionIds.includes(election.electionId))
+    .filter((election) => election.status?.toLowerCase() === 'open');
+
+  return (
+    <div style={styles.voteInfo}>
+      <div>
+        {!adminPage && !allOpenElections && !readOnly && (
+          <Alert
+            id={'vote-disabled-alert'}
+            description={'Voting is disabled since this election is not open.'}
+            title={'Voting is disabled since this election is not open.'}
+            type={'danger'}
+          />
+        )}
+      </div>
+      <div>
+        <CollectionSubmitVoteBox
+          votes={currentUserVotes}
+          isFinal={isChair}
+          isDisabled={adminPage || readOnly || isEmpty(currentUserVotes) || !allOpenElections}
+          isApprovalDisabled={isApprovalDisabled}
+          isLoading={isLoading}
+          adminPage={adminPage}
+          bucketKey={convertLabelToKey(get(bucket, 'key', 'collection-submit-vote-box'))}
+          updateFinalVote={updateFinalVote}
+          reloadFn={reloadFn}
+        />
+      </div>
+    </div>
+  );
+};
+
+// Main Component
+export default function MultiDatasetVoteSlab({
+  title,
+  bucket,
+  collection,
+  dacDatasetIds,
+  isChair,
+  isApprovalDisabled,
+  isLoading,
+  readOnly,
+  adminPage,
+  updateFinalVote,
+  reloadFn
+}: MultiDatasetVoteSlabProps) {
+  const [currentUserVotes, setCurrentUserVotes] = useState<Vote[]>([]);
+  const [dacVotes, setDacVotes] = useState<Vote[]>([]);
+  const [isDMI, setIsDMI] = useState(false);
+  const { algorithmResult } = bucket;
+  const getMemberVoteSectionTitle = () => {
+    if (adminPage) return 'DAC Member Votes';
+    if (isChair) return 'My DAC Member\'s Votes (detail)';
+    return 'Other DAC Member\'s Votes';
+  };
+
+  useEffect(() => {
+    const sorted = Object.values(collection.dars).sort(
+      (a, b) => new Date(b.submissionDate).getTime() - new Date(a.submissionDate).getTime()
```

Comment:
We're sorting by `id` elsewhere in the frontend if you're looking for consistency. ID is always increasing.

Reply:
ID does also work. I specifically looked for this and found that we're using submission date on the back end: https://github.com/DataBiosphere/consent/blob/fe01d187568443679cb9b49718e05892f11314c5/src/main/java/org/broadinstitute/consent/http/models/DarCollection.java#L135